applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -189,6 +189,69 @@ For an example of a bootloader Kconfig configuration file defined by the applica
   Both mentioned firmware upgrade methods are not used simultaneously by any of the configurations.
   For example, the ``nrf52840dk/nrf52840`` board in ``mcuboot_smp`` file suffix uses only the background DFU and does not enable the serial recovery feature.
 
+RAM load mode
+~~~~~~~~~~~~~
+
+The RAM load mode is used for the :ref:`background DFU <nrf_desktop_bootloader_background_dfu>`.
+In this mode, the MCUboot bootloader copies the application image from the non-volatile memory (NVM) to the RAM and boots it from there.
+The image, built for the RAM address space, can be stored in one of the MCUboot slots located in the NVM.
+During the DFU process, the oldest image is overwritten with the new image.
+The new image is then booted from the RAM.
+
+.. warning::
+   The RAM load mode of the MCUboot bootloader is not officially supported in |NCS|.
+   However, the mode is available in the |NCS| as the support for this feature has been developed as part of the Zephyr RTOS project.
+   This feature is only used in the limited context for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target configuration to improve USB performance.
+
+The RAM load mode of the MCUboot bootloader can be used to speed up the code execution for the application image, as code execution from the RAM is faster than from the NVM.
+This particular benefit can help to improve the device performance during the activities that require high CPU usage.
+As an example, the nRF Desktop application uses the RAM load mode in the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target configuration to achieve 8 kHz report rate over USB.
+
+To set the MCUboot mode of operations to the RAM load mode, enable the :kconfig:option:`SB_CONFIG_MCUBOOT_MODE_RAM_LOAD` Kconfig option in the sysbuild configuration.
+
+To support the RAM load mode, you must use DTS as the partitioning method, as the Partition Manager (PM) is not supported in this mode.
+To satisfy this requirement, disable explicitly the :kconfig:option:`SB_CONFIG_PARTITION_MANAGER` Kconfig option in your sysbuild configuration.
+Additionally, you must define the custom memory layout for the RAM in your target board configuration.
+Your RAM layout must define the following DTS child nodes as part of the ``cpuapp_sram`` DTS node in the address order listed below:
+
+* ``cpuapp_sram_app_rxm_region`` - This DTS node defines the hard limits for the executable ROM section (with the application image) and must be aligned with the :kconfig:option:`CONFIG_BOOT_IMAGE_EXECUTABLE_RAM_START` and the :kconfig:option:`CONFIG_BOOT_IMAGE_EXECUTABLE_RAM_SIZE` Kconfig options.
+* ``cpuapp_sram_mcuboot_ram_region`` - This DTS node must be assigned to the MCUboot image as its chosen SRAM DTS node.
+
+For an example of the custom RAM layout that satisfies these requirements, see the :file:`nrf/applications/nrf_desktop/configuration/nrf54lm20dk_nrf54lm20a_cpuapp/memory_map_ram_load.dtsi` file.
```

Comment:
Consider mentioning that the memory map must be shared among images (MCUboot + application)